Doc/library/math.rst

```diff
@@ -144,8 +144,7 @@ Number-theoretic functions
 
 .. function:: factorial(n)
 
-   Return *n* factorial as an integer.  Raises :exc:`ValueError` if *n* is not integral or
-   is negative.
+   Return factorial of the non-negative integer *n*.
```

Comment:
@sobolevn, I'm not sure if this is a good wording.  See e.g. https://github.com/python/cpython/pull/101492#discussion_r1096791486

Reply:
oh, ok :)